nd4j/nd4j-common/src/main/java/org/nd4j/util/FingerPrintKeyer.java

```diff
@@ -1,35 +1,18 @@
-/*
-
-Copyright 2010, Google Inc.
```

Comment:
This file actually is under Google's copyright, we can't just replace it with our own!

Reply:
Oh, missed that! Thanks.

Reply:
Fixed.